info.yml

```diff
@@ -74,6 +74,11 @@ configuration:
         description: Optionally choose to use 'Sgtk' as the primary menu name instead of 'Shotgun'
         default_value: false
 
+    launch_builtin_plugin:
+        type: str
+        description: Comma-separated list of tk-maya plugins to load when launching Maya
```

Comment:
maybe it's worth specifying here that as soon a value is specified, the 'classic' launching is turned off?